src/audio.rs

```diff
@@ -0,0 +1,7 @@
+use bevy::prelude::*;
+
+#[derive(Component, Default)]
+pub struct BackgroundMusic;
+
+#[derive(Component, Default)]
+pub struct SoundEffect;
```

Comment:
These are really nice! I'd appreciate some comments guiding users why they should care about them and what they might be used for. Especially a `Trigger<OnAdd, SoundEffect>` and similar would imo be quickly useful.